chatterbot/trainers.py

```diff
@@ -377,7 +377,7 @@ def train(self):
 
                 for row in reader:
                     if len(row) > 0:
-                        text = row[3]
+                        text = unicode(row[3], errors='ignore')
```

Comment:
test are failing due to 

```
lint runtests: commands[0] | flake8

./chatterbot/trainers.py:380:32: F821 undefined name 'unicode'

ERROR: InvocationError: '/home/travis/build/gunthercox/ChatterBot/.tox/lint/bin/flake8'
```

Instead use convert_to_ascii method to do

https://github.com/gunthercox/ChatterBot/blob/bdc640b30c4ed228520c70b8ae01b92f83eddba9/chatterbot/preprocessors.py#L44